### PR TITLE
T2 servicefabricmanagedclusters 2021 02 26

### PR DESCRIPTION
**Pull request by zhangyan133**
https://github.com/Azure/sdk-release-request/issues/1297
--

### CODE REVIEW

File: sdk/servicefabricmanagedclusters/azure-mgmt-servicefabricmanagedclusters/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Release History
+
+## 0.0.0 (2021-02-26)
```
Comment (00Kai0) [inline sdk/servicefabricmanagedclusters/azure-mgmt-servicefabricmanagedclusters/CHANGELOG.md:3]: remove this `0.0.0` changelog

File: sdk/servicefabricmanagedclusters/azure-mgmt-servicefabricmanagedclusters/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+change_log.py: error: the following arguments are required: base, latest
+
+## 0.1.0 (1970-01-01)
```
Comment (00Kai0) [inline sdk/servicefabricmanagedclusters/azure-mgmt-servicefabricmanagedclusters/CHANGELOG.md:7]: it should be `## 1.0.0b1 (2021-02-26)`

File: sdk/servicefabricmanagedclusters/azure-mgmt-servicefabricmanagedclusters/azure/mgmt/servicefabricmanagedclusters/_version.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+VERSION = "0.0.0"
```
Comment (00Kai0) [inline sdk/servicefabricmanagedclusters/azure-mgmt-servicefabricmanagedclusters/azure/mgmt/servicefabricmanagedclusters/_version.py:9]: It should be `1.0.0b1`

**Review by 00Kai0 — CHANGES_REQUESTED** [review]

**Review by 00Kai0 — APPROVED** [review]

**Review by msyyc — APPROVED** [review]

**Pull request merged**